src/Shell.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Laravel\Tinker;
+
+use ErrorException;
+use Exception;
+use Illuminate\Support\Str;
+use Psy\CodeCleaner\NoReturnValue;
+use Psy\Exception\BreakException;
+use Psy\Exception\Exception as PsyException;
+use Psy\Shell as BaseShell;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class Shell extends BaseShell
+{
+    const BUFF_PROMPT = '  ';
+    const PROMPT = '> ';
+
+    /**
+     * The OutputStyle instance.
+     *
+     * @var \Symfony\Component\Console\Style\SymfonyStyle
+     */
+    protected $output;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function doRun(InputInterface $input, OutputInterface $output): int
+    {
+        return parent::doRun($input, $this->output = new SymfonyStyle($input, $output));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getHeader(): string
+    {
+        return '';
```

Comment:
I'd prefer we keep the header. As @Wirone pointed out, this is the only indication to Tinker users that what they're using is actually (literally) 99% PsySH, with a bit of a shim to make it work in their app.

Reply:
I believe we should apply such exposing of original tool not only for tinker/PsySH, but also for pint/PHP CS Fixer

Reply:
And any other tools in similar case